### ISSUE TITLE
Long messages are duplicated

### ISSUE THREAD

**Issue by PKEv** [qTox/qTox#1703]
When you send a long message it duplicated in the history of sending messages.
Only at the sender side
sample text:
"qrencode

Download the qrencode from http://fukuchi.org/works/qrencode/ or direct from https://code.google.com/p/qrencode-win32/source/checkout , build project "..\qrencode-win32\vc8\qrcodelib\", you must copy files from release in: "qrcodelib.dll" to \qTox\libs\bin\qrcodelib.dll"; "qrencode.h" to \qTox\libs\include\qrencode.h"; "qrcodelib.lib" to "\qTox\libs\lib\qrencode.lib" with rename!!!

Setting up Path

Add MinGW/MSYS/CMake binaries to the system path to make them globally accessible. Open Control Panel -> System and Security -> System -> Advanced system settings -> Environment Variables... In the second box search for the PATH variable and press Edit... The input box "Variable value:" should already contain some directories. Each directory is separated with a semicolon. Extend the input box by adding ";C:\MinGW\bin;C:\MinGW\msys\1.0\bin;C:\Program Files (x86)\CMake 2.8\bin". The very first semicolon must only be added if it is missing. CMake may be added by installer automatically.

Cloning the Repository

Clone the repository (https://github.com/tux3/qTox.git) with your preferred Git client. SmartGit is very nice for this task (you may need to add the path to the git.exe system variable Path). The following steps assume that you cloned the repository at "C:\qTox". If you decided to choose another location, replace corresponding parts.

Getting dependencies

Run bootstrap.bat in cloned C:\qTox directory Script will download rest of dependencies compile them and put to appropriate directories."

**Comment by SkyzohKey** [comment]
Same for me :/

**Comment by zetok** [comment]
isn't that a duplicate of #1375 ?

**Comment by PKEv** [comment]
No, this happend is only on the sender side and the message is fully duplicated

**Comment by zetok** [comment]
@PKEv this is exactly what #1375 is about.

Edit: or maybe not so exactly. It also includes problem with message splitting, beside text being duplicated on sender side.

**Comment by agilob** [comment]
That's interesting, those long messages are saved only once to database.
